packages/format-library/src/link/index.js

```diff
@@ -20,6 +20,8 @@ import InlineLinkUI from './inline';
 
 const name = 'core/link';
 
+const isEmail = ( email ) => /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/.test( email );
```

Comment:
This function should probably be added to the `@wordpress/url` package.

Reply:
The matcher is there already, actually: https://github.com/WordPress/gutenberg/blob/master/packages/url/src/index.js#L7

Reply:
Sure.

Reply:
Added to `@wordpress/url` in https://github.com/WordPress/gutenberg/pull/14857/commits/cb8272af748480b7c6db0cff64154c9e16dcf958 and fixed in  https://github.com/WordPress/gutenberg/pull/14857/commits/6620026a0cc79af5d37b776dc634467cf27d613f.